### ISSUE TITLE
NZB name truncate file name extension (GUI)

### ISSUE THREAD

**Issue by zuricarmi** [mbruel/ngPost#169]
Hi
In windows latest ngpost ver wene I post the files
a.r01
a.r02
a.r03 ...
I get nzb files a.nzb, a_1.nzb, a_2.nzb ...
Is there a way for nzb files to get full name of the file with .nzb at the end?
like
a.r01.nzb, a.r02.nzb, a.03.nzb ...
Thanks

**Comment by mbruel** [comment]
yeah that's normal.
you're supposed to put all the archive on a same nzb that will be called a.nzb
otherwise you can rename manually the nzb filename on the bottom left (nzb file) to re-add the extension.
hope this help ;)

**Comment by Abu3safeer** [comment]
I agree with this, though I use command line and I can do it with -O "video.mkv.nzb" 
but I really wanted this to work just out of the box.
